### DISCUSSION TITLE
Add a 100% button to the toolbar

### DISCUSSION THREAD

**Discussion by giwul70** [sumatrapdfreader/sumatrapdf#4095] [Ideas]
Currently there is an increase/decrease button in the toolbar.

Suggestion: add a 100% button in between.

It is quicker than File > Zoom > scroll to 100%.

Thanks and .. keep up the good work!

![SumatraPDF-Suggestion-Add 100 pct button-18022024 052637](https://github.com/sumatrapdfreader/sumatrapdf/assets/101186455/3161f375-8f51-4a92-8b22-7637a394760a)

**Comment by GitHubRulesOK** [comment]
there are too many commands to add to the console so many have 2 keys as keyboard would not be enough icons 

you can change the default CTRL+1 to a single key by customising your choice of key for `CmdZoomActualSize` but the rolling `Z` key  is a good way to full zoom between the other 3 states so perhaps `X` is a good choice.

**Comment by giwul70** [comment]
Thank you very much again.

I have confess: I use SumatraPDF 'as it is', so, all defaults, Never went into these 'advanced' matters.
So, I had to figure out how/where to use these Cmd's :-)

Found it and edited the shortcuts in SumatraPDF-settings.txt

```
Shortcuts [
	[
		Cmd = CmdZoomActualSize
		Key = x
	]
]
```

Worked out fine, **thanks** again for the tip.

**Comment by kjk** [comment]
You can now add buttons to toolbar. See https://www.sumatrapdfreader.org/docs/Customize-toolbar
